types/tiny-slider-react/index.d.ts

```diff
@@ -0,0 +1,264 @@
+// Type definitions for tiny-slider-react 0.3
+// Project: https://github.com/jechav/tiny-slider-react#readme
+// Definitions by: Christian Rackerseder <https://github.com/screendriver>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.6
+import * as React from "react";
+
+interface CommonOptions {
+    /**
+     * Default: 0.
+     * The initial index of the slider.
+     */
+    startIndex?: number;
+    /**
+     * Default: 1.
+     * Number of slides being displayed in the viewport.
+     */
+    items?: number;
+    /**
+     * Default: 0.
+     * Space between slides (in "px").
+     */
+    gutter?: number;
+    /**
+     * 	Default: 1.
```

Comment:
It is better to improve the format

Reply:
True. I blindly copied all that stuff from the official docs 😉 
It is fixed now.